packages/web/src/dev/FullStack.tsx

```diff
@@ -221,16 +222,22 @@ class FullStack extends React.PureComponent<I18nProps & ScreenProps, State> {
               <Li style={textStyles.invert}>{t('proof.onChain')}</Li>
             </StackSection>
             <View
+              // @ts-ignore
```

Comment:
why dis?

Reply:
was an issue with using transition as a property , fixed it up now